From 0672eebb472aaef8449aec4cb064acc86e3aae37 Mon Sep 17 00:00:00 2001
From: Hector Martin <marcan@marcan.st>
Date: Thu, 30 Jun 2022 21:48:46 +0900
Subject: [PATCH 033/171] arm64: dts: apple: t600*: Add t8103 compat to cpufreq
 node

t600x seems to be fully compatible with t8103, but t8112 introduced
minor changes which means we can't provide full functionality with the
common compatible. Let t600x claim t8103 compatibility, treating the
latter as the common baseline for this SoC family.

Signed-off-by: Hector Martin <marcan@marcan.st>
---
 arch/arm64/boot/dts/apple/t600x-die0.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/apple/t600x-die0.dtsi b/arch/arm64/boot/dts/apple/t600x-die0.dtsi
index b24dd6ed0a37..4c203778aa89 100644
--- a/arch/arm64/boot/dts/apple/t600x-die0.dtsi
+++ b/arch/arm64/boot/dts/apple/t600x-die0.dtsi
@@ -7,7 +7,7 @@
  */
 
 	cpufreq_hw: cpufreq@210e20000 {
-		compatible = "apple,t6000-soc-cpufreq", "apple,soc-cpufreq";
+		compatible = "apple,t6000-soc-cpufreq", "apple,t8103-soc-cpufreq", "apple,soc-cpufreq";
 		#freq-domain-cells = <1>;
 	};
 
-- 
2.34.1

